src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -780,12 +780,12 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
 
     void ConptyConnection::StartInboundListener()
     {
-        THROW_IF_FAILED(CTerminalHandoff::s_StartListening(&ConptyConnection::NewHandoff));
-    }
+        static const auto init = []() noexcept {
+            CTerminalHandoff::s_setCallback(&ConptyConnection::NewHandoff);
+            return true;
+        }();
 
-    void ConptyConnection::StopInboundListener()
```

Comment:
never called?

Reply:
I missed your comment. Yeah it's never called and removing it allows us to ensure it remains safe (the registration gets removed directly in the callback routine).